test/__snapshots__/mathml-spec.js.snap

```diff
@@ -112,6 +110,35 @@ exports[`A MathML builder should generate the right types of nodes 1`] = `
 
 `;
 
+exports[`A MathML builder should generate the right types of nodes 2`] = `
+
+<math>
+  <semantics>
+    <mrow>
+      <mi>
+        sin
+      </mi>
+      <mo>
+        ⁡
+      </mo>
```

Comment:
Is this `<mo>` supposed to contain a `&#x2061;`?

Reply:
I believe it already is.  Shame that we can't see it in the preview, but I checked there's something there.